app/models/group/federation.rb

```diff
@@ -86,6 +86,7 @@ class DispatchAddress < Jubla::Role::DispatchAddress
 
   class ItSupport < Jubla::Role::Member
     self.permissions = [:impersonation]
+    include Group::UniqueNextcloudGroup
```

Comment:
Das ist vermutlich schon durch die gleiche Inklusion in `Jubla::Role::Member` vorhanden, und daher hier unnötig.